docs/src/code-examples/generate-template-preview/node.mdx

```diff
@@ -0,0 +1,43 @@
+---
+lang: node
+---
+
+```javascript
+const personalisation = { 'foo': 'bar' };
```

Comment:
@andrewoh531 might want to check that all the JS stuff is running through prettier. I think prettier runs code fences inside MDX too... e.g. in this the brackets below are redundant and a bit noisy.

Also, I wonder if it's better to use `async/await` instead of `then/catch`?

Reply:
Thanks I've ran the prettier.

Regarding `async/await` I'm kinda tempted to keep it for brevity/readability:

```
client 
  .previewTemplateById(templateId, personalisation)
  .then(response => console.log(response))
  .catch(err => console.error(err))
```
vs
```
try {
  const resp = await client.previewTemplateById(templateId, personalisation)
  console.log(resp);
} catch(e) {
  console.error(err)
}
```
Not a big diff but just in terms of what's going on overall. Also I think most `js` devs would/should know of `async/await` and `promises`.